### PR TITLE
ci(chromatic): hardcode diffThreshold to 0.7

### PR DESCRIPTION
**Pull request by benelan**
# Edit

Alright, this is ready for review now. I set the diffThreshold to 0.7 which is higher than I had hoped, but it seems to be working. We need to keep an eye out for a bit to make sure actual changes are caught, but it should be fine. We are still getting false positives from the floating-ui components, but [specifying the viewport](https://www.chromatic.com/docs/viewports) may help. I'll work on that in a separate PR.

---

**Related Issue:** #

## Summary
I bumped the GH Secret up to `1` and we still got diffs. I'm hoping its just something with the Secret -> Env Var pipeline that's causing issues.

To make sure I'm barking up the right tree, I bumped up the diff threshold for slider and date-picker to the most relaxed. I also added the threshold per story for slider, because that is how they show it documented:
https://www.chromatic.com/docs/threshold

I'm hoping that date-picker passes as well so I don't have to go through and add it to every story lol. 

I've also been trying to save snapshots by checking real PRs for the chromatic results, but I'm just going to rerun it in this PR until I get it working. We have plenty of snapshots left and its the end of the month.

### CODE REVIEW

File: src/components/slider/slider.stories.ts [hunk 1]
```diff
@@ -158,8 +155,8 @@ export const rangeLabeledTicksEdgePositioningAtMin_TestOnly = (): string => html
   ></calcite-slider>
 `;
 
-export const Histogram = (): HTMLCalciteSliderElement => {
-  const slider = document.createElement("calcite-slider") as HTMLCalciteSliderElement;
+export const Histogram = (): any => {
```
Comment (driskull) [inline src/components/slider/slider.stories.ts:158]: Why the change to type as `any` in these cases?

File: src/components/slider/slider.stories.ts [hunk 2]
```diff
@@ -186,37 +183,8 @@ export const Histogram = (): HTMLCalciteSliderElement => {
   return slider;
 };
 
-export const HistogramWithColors = (): HTMLCalciteSliderElement => {
```
Comment (driskull) [inline src/components/slider/slider.stories.ts:189]: Did we need to remove this story?

**Review by driskull — APPROVED** [review]
👍

File: src/components/slider/slider.stories.ts (hunk 1 above)
Comment (benelan) [reply src/components/slider/slider.stories.ts:158]: We don't have access to the component types from the stories so it throws linting errors
image

I'm pretty sure I tried importing `components.d.ts` at one point, which fixed the linting errors but caused build errors. I can try to dig that up again when I have some time

Comment (benelan) [reply src/components/slider/slider.stories.ts:158]: Yeah it was in #5482

Here was the error that popped up when importing `components.d.ts`:
https://github.com/Esri/calcite-components/actions/runs/3275079841/jobs/5389458993#step:6:50

**Review by alisonailea — APPROVED** [review]
